README.md

```diff
@@ -19,6 +19,9 @@
   <a href="https://github.com/developmentseed/geojson-pydantic/blob/master/LICENSE" target="_blank">
       <img src="https://img.shields.io/github/license/developmentseed/geojson-pydantic.svg" alt="Downloads">
   </a>
+  <a href="https://conda.anaconda.org/conda-forge" target="_blank">
```

Comment:
can we put the link to https://anaconda.org/conda-forge/geojson-pydantic directly here?